package.json

```diff
@@ -11,17 +11,18 @@
     "@types/react-dom": "^17.0.9",
```

Comment:
Please do not add useless files to your PR.
You can use **git add src/** command to add needed files
**git status** command — for checking file status